www/src/public/js/controllers/builder/main.js

```diff
@@ -926,30 +942,37 @@
 
             var importStr = $scope.importModel.input;
             if (importStr) {
-                var listVersion = -1;
+                var listVersion = 3;
 
-                // check if list has a version number
-                var asteriskIdx = listCookieStr.indexOf("*");
-                var tildeIdx = listCookieStr.indexOf("~");
-                if (asteriskIdx < tildeIdx) {
+                // check if list has a version number attached to string.  If so, set listVersion
+                // the -1 check is here get around lists that do not have a list number reporting as a -1 which otherwise causes this function to always run and remove the first letter of lists that do not have a version number
+                var asteriskIdx = importStr.indexOf("*");
+                var tildeIdx = importStr.indexOf("~");
+                if (asteriskIdx < tildeIdx && asteriskIdx != -1) {
                     listVersion = Number(importStr.substring(0, asteriskIdx));
                     importStr = importStr.slice(asteriskIdx);
                 }
 
                 var listStrs = importStr.split("*").filter(function(el) {return el.length != 0});;
                 for (let i = 0; i < listStrs.length; ++i) {
                     var newList;
-                    if (listVersion > 0) {
+                    // if list version is higher than 3 it should always have a version number, thus use the listVersion set above.  Otherwise, cycle through the previous versions until it works.
+                    if (listVersion > 3) {
```

Comment:
again, this can be left at `> 0` because all this was checking was "is the list version included in the import string". your check here would also fail since you'd want to check `>= 3` rather than `> 3`. We released version 3 and cln at the same time.

I believe the changes in this for loop aren't needed.